### PR TITLE
feat(FEC-10057): move the plugin manager to kaltura player

### PR DESCRIPTION
**Pull request by yairans**
### Description of the Changes

import the plugin infra and types from kaltura-player
Depends on https://github.com/kaltura/kaltura-player-js/pull/332

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -62,6 +62,7 @@
     "flow-bin": "^0.129.0",
     "husky": "^4.2.5",
     "istanbul": "^0.4.5",
+    "kaltura-player-js": "https://github.com/kaltura/kaltura-player-js.git#FEC-10057",
```
Comment (RoyBregman) [inline package.json:65]: is this correct with FEC-10057?

Comment (yairans) [reply package.json:65]: it's just to pass travis. anyway it's peer-dependency so it's still correct

**Comment by yairans** [comment]
@OrenMe I've added .idea/runConfigurations/karma_conf_js.xml on purpose to add `NODE_ENV` when testing via karma.
I use this env variable in webpack.config.js

**Comment by yairans** [comment]
I can

**Pull request merged**